RiotSharp/Misc/Platform.cs

```diff
@@ -57,6 +57,11 @@ public enum Platform
         /// <summary>
         /// Korea.
         /// </summary>
-        KR
+        KR,
+
+        /// <summary>
+        /// No Platform (e.g. bots).
```

Comment:
It might be nice to clarify this e.g `platformId of bot players`

Reply:
Updated the comment.